### DISCUSSION TITLE
how to properly filter ec2 instance import using filter by vpc?

### DISCUSSION THREAD

**Discussion by sebanazarian** [GoogleCloudPlatform/terraformer#1179] [Q&A]
how to properly filter ec2 instance import using filter by vpc?

**Comment by adamkamali** [comment]
none of the filters seem to work except for tags. 

The only option is to go and tag all resources in your VPC. 

I used this to list all "ec2 instance" within the VPC. 

`aws ec2 describe-instances --filters Name=vpc-id,Values=vpc-d0c675b4     --query 'Reservations[*].Instances[*].{Instance:InstanceId,Subnet:SubnetId}' `

**Reply by openPablo** [reply]
It seems like filtering is completely broken ?
